docs/en/query_language/select.md

```diff
@@ -448,6 +448,19 @@ The table names can be specified instead of `<left_subquery>` and `<right_subque
 
 See the standard [SQL JOIN](https://en.wikipedia.org/wiki/Join_(SQL)) description.
 
+#### Syntax Limitations
+
+For multiple `JOIN`:
+
+- You cannot perform `SELECT * ...` if you use `SELECT` subqueries for joining. Using of `*` is available only for joining tables.
```

Comment:
Cryptic. I  don't understand that.

Reply:
@4ertus2. Let's clarify this statement with an example. Can you come up with something? Maybe some clarification words are required?

Reply:
Taking all columns via `*` is available only if tables are joined, not subqueries.